core/templates/dev/head/pages/exploration_editor/ParameterMetadataService.js

```diff
@@ -90,33 +76,23 @@ oppia.factory('ParameterMetadataService', [
       ExpressionInterpolationService.getParamsFromString(
         state.content.getHtml()).forEach(
         function(paramName) {
-          result.push({
-            action: PARAM_ACTION_GET,
-            paramName: paramName,
-            source: PARAM_SOURCE_CONTENT
-          });
+          result.push(ParamMetadataObjectFactory.createWithGetAction(
+            paramName, PARAM_SOURCE_CONTENT));
```

Comment:
Would it be better to pass in null for the last argument explicitly? Would this break anything?

Reply:
Without passing in `null` , sourceInd is `undefined`. I think it is better to pass in `null` explicitly as last argument.